main.go

```diff
@@ -120,6 +121,7 @@ func main() {
 		recipeCmd.Flags().BoolVar(&contenderEnabled, "contender", false, "spam nodes with contender")
 		recipeCmd.Flags().StringArrayVar(&contenderArgs, "contender.arg", []string{}, "add/override contender CLI flags")
 		recipeCmd.Flags().StringVar(&contenderTarget, "contender.target", "", "override the node that contender spams -- accepts names like \"el\"")
+		recipeCmd.Flags().StringArrayVar(&prefundedAccounts, "prefunded-account", []string{}, "Fund this account in addition to static prefunded accounts, the input should the account's private key in hexadecimal format prefixed with 0x, the account is added to L1 and to L2 (if present)")
```

Comment:
Looks like we can provide multiple accounts here. So maybe the flag is `add-prefunded-accounts` or `prefund-accounts` and we can document that possibility ("fund _these accounts_").

Reply:
Hmm, I see. I think it's fine if we treat `contender.arg` as an exception. There is still value in making it obvious to the users that providing multiple accounts is already possible. Does that make sense?